### ISSUE TITLE
Bug with sim mods in the unit creation dialog

### ISSUE THREAD

**Issue by Garanas** [FAForever/fa#5388]
**Describe the bug**

See also: https://forum.faforever.com/topic/6422/developers-iteration-iii-of-2023/109

![image](https://github.com/FAForever/fa/assets/15778155/eb801001-50a2-41bd-be5c-bd4bf612fcc6)

**To Reproduce**

Start a game with sim mods that add units.

**Comment by Garanas** [comment]
@HUSSAR-mtrela do you have time to pick this up?

**Comment by HUSSAR-mtrela** [comment]
@Garanas  yes, I have time to fix. Is this issue occurring with any SIM mods that add units or maybe it is caused by issue with localizing mod names with Chinese/Korean language?

**Comment by Garanas** [comment]
Good question, I don't know. I noticed that it works fine for BlackOps in the english language

**Comment by HUSSAR-mtrela** [comment]
this issue is fixed in above PR

**Comment by MrRowey** [comment]
@Garanas @HUSSAR-mtrela 
![image](https://github.com/FAForever/fa/assets/20344151/3007c939-cd10-4ae5-b4b2-9388955397c6)
As you can see in the image above my Sim Mod shows as [no text]
